jenkins/gradle/gradle-check.jenkinsfile

```
@@ -102,10 +107,9 @@ pipeline {
 
                     def agent_name_array = params.AGENT_LABEL.tokenize('-')
                     def agent_name = agent_name_array[2] + " " + agent_name_array[4]
-                    def CREDENTIAL_ID = 'jenkins-production-dockerhub-credential-readonly'
 
                     echo("Build Cause: ${BUILD_CAUSE}")
-                    withCredentials([usernamePassword(credentialsId: CREDENTIAL_ID, usernameVariable: 'DOCKER_USERNAME', passwordVariable: 'DOCKER_PASSWORD')]) {
+                    withSecrets(secrets: secret_dockerhub_readonly){
```

If secrets are just used once, would recommend to not use variable. If helps to read the code better and maintenance as well.
```suggestion
                    withSecrets(secrets: [
    [envVar: 'DOCKER_USERNAME', secretRef: 'op://opensearch-infra-secrets/dockerhub-production-readonly-credentials/username'],
    [envVar: 'DOCKER_PASSWORD', secretRef: 'op://opensearch-infra-secrets/dockerhub-production-readonly-credentials/password']
]){
```

---

1password does not recommend that way.

They asked to either use a var or use     environment {} block so it is cleanly organized.

Before we are very familiar with this new tool, I would suggest we just follow the guidelines as it is easier to maintain.

Thanks.